### PR TITLE
Pretext map

### PR DESCRIPTION
**Pull request by astrovsky01**
extra file

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)
Add pretext map, and include in the pretext collection

### CODE REVIEW

File: tools/pretext/pretext_map.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <param name="map_qual" type="integer" label="Minimum mapping quality:" min="0" optional="true"/>
+        <conditional name="filter">
+            <param name="filter_type" type="select" label="How to filter: ">
+                <option value="" selected="true">No filter</option>
+                <option value="in">Filter to include only specified sequence names</option>
+                <option value="out">Filter to exclude specified sequence names</option>
+            </param>
+            <when value=""/>
+            <when value="in">
+                <param name="filter_list" type="text" label="Filter list" help="Comma separated list of sequence names to use as a filter" value="seq_1, seq_2"/>
```
Comment (bgruening) [inline tools/pretext/pretext_map.xml:66]: do we need a validator here? are spaces allowed? only alphanumeric + comma?

Comment (astrovsky01) [reply tools/pretext/pretext_map.xml:66]: It's just the ref names, so mostly that and underscores, which I thought were allowed by default

Comment (bgruening) [reply tools/pretext/pretext_map.xml:66]: They are allowed. But we want to disallow certain inputs in free-text fiels. E.g. something like that `' && rm -rf / &&'`

Comment (bgruening) [reply tools/pretext/pretext_map.xml:66]: The more we restrict the safer we are. If you think only underscores and text, than lets allow only those.

Comment (astrovsky01) [reply tools/pretext/pretext_map.xml:66]: Sounds good. One sec

**Review by bgruening — APPROVED** [review]

**Pull request merged**